### ISSUE TITLE
Idea: Increasing prices

### ISSUE THREAD

**Issue by garrynewman** [Facepunch/RustWorkshop#94]
Items all go for sale for $1. After being a week old the price is $2. The price increases every week until they reach $20. They're then removed from the item store - as more people will be buying from the marketplace.

Goals:

- Emphasis on newly released skins
- More people buy new skins
- Skins naturally transition to the marketplace at a decent price

Feedback?

**Comment by Vazul267** [comment]
It would be easily profitable for people on the market.

**Comment by CapiMachja** [comment]
i think it's a great idea. on a slightly unrelated note, could you make it so we can sort skins from price ? would be very much appreciated :)

**Comment by East113** [comment]
I agree completely with the .99 starting price. Maybe cap the buy time to 4 weeks until it is removed, or perhaps double the price every week, then after $16 cap it at $20. Will promote a more hasty time frame to buy soon, and will be a quicker time frame to get off  the market.  All in all though, loving the idea.

**Comment by raycohaex** [comment]
thanks garry <3

**Comment by raycohaex** [comment]
But i think many people are still unaware of the rust shop, concidering some people buy skins for double price or even 30$ when its only 99 cents

**Comment by East113** [comment]
That could also be people transferring money to friends through steam, I don't know of any other way to send money on steam. Just a guess though.

**Comment by Refused9150** [comment]
I love the idea

**Comment by raycohaex** [comment]
My only feedback would be, maybe there should be 3 or more/less cathegory's. based on quality of the skin/expectations etc. Here is a shitty made example. But idk if it's a good idea 

*edit
pixel mario = cathegory 1
2d mario = cathegory 2
3D realistic mario = cathegory 3

and prices are just an example
![image](https://cloud.githubusercontent.com/assets/11200658/21701765/e914eb4e-d3a7-11e6-8ec9-394412c3e2e5.png)

**Comment by partyboycs** [comment]
Love it Garry, 10/10. Let's all make some money off these skins :D You could even just remove some of the skins after a couple weeks which would make them rare and valuable :)

**Comment by dice4321** [comment]
does he mean all items like the tempered ak and such, or all new items?

**Comment by joeytee** [comment]
I like the idea a lot, perhaps you could do just one premium priced skin each week though? (as well as the cheap ones) perhaps around the £3-£5 mark.

**Comment by v1be** [comment]
Anything that avoids price stagnation of older items is a plus. It kills in-game economies when old items are worth nothing. As a trader since early TF2/ CS I'm optimistic and think you're handling the economy well, and asking all the right questions.  I also like that you have some completely retired items such as the new Christmas ones - perhaps you could add a category that shows such items exist, to keep things interesting economy-wise.

**Comment by leandrosantos23** [comment]
nice idea, plz make this

**Comment by Subza** [comment]
I agree with DutchAllians suggestion. Having all skins start at the same price is a bit strange considering some are simply better looking/will be used more than others.

**Comment by dcluniecrm** [comment]
personally and I've found through friends buying skins, is that regardless of the "quality" if the skin is cheap they'll buy it.  So I'd say go for a mobile app mentality, charge for $1 for everything, most people will just buy it to have it, e.g. i'm kind of bored with skin a, lets change it up this week with skin b, vs seeing a skin for $10 and then saying, $10 bucks for skin a, no way i'm paying $10 for a skin, hell i got the game on sale for 9.99!  That's crazy, and then they assuming all skins are useless and pricy never returning to the store to buy ANY skins. just my two cents.

**Comment by StabbyMcStabface** [comment]
I was thinking about this last week when I saw you add all of the new skins at .99 cents. I assumed this was going to be something to offset more "premium" priced skins. I started thinking that a tiered pricing strategy might be interesting, with say Common, Limited, and Rare (>500?) items. 

Commons range from .99 to 1.99, Limited  2.99 to 4.99, and Rare 9.99-14.99. The rarest items would be either exceptionally well done or just really smart/funny concepts that will appeal to collectors. In any event, the stock stops selling after however many weeks and then it goes to market and is replaced by a whole new round of goods. 

The other idea I had was more complicated and that involved having pricing reflect the item's "in-game" status (ie - Entry level items, mid-tier, and endgame) but then I just confused myself and ended up watching porn.

**Comment by hersic** [comment]
Skins should be priced by how much the item they are for is used in-game by players. Things like burlap clothing, hide clothing, and other low protection clothes should be priced lower than the other min-maxing clothing, so they have a chance to sell too since they won't be as crafted as the best stuff. The skins for the most crafted items high end stuff will sell even if they're priced higher, as seen previously on the most sold store tab.

Also, I believe that the skins removed should be in the order of the date added, there's still plenty of skins from late 2015-early 2016 in the store. 

Another thing I want to add is that it'd be more motivational for workshop authors if you made a list of item categories to make skins for every week, and then search the given categories in the workshop and pick one out of each on Thursday. A lot of authors feel that their work isn't seen, and this way they'd at least know that it wasn't good enough and they should try again next week.

**Comment by hersic** [comment]
Adding more filters to Sort by other than Most Popular (Today) and Most Recent would help good skins from being lost in the workshop.
I suggest adding Most popular (Past week), (Past Month), (Past Year), (All time) filters, and using Yes %, favorites and views numbers for sorting.

**Comment by StabbyMcStabface** [comment]
Yeah, what hersic said. LOL. I'm also concerned about "old items". I'm a bit obsessive once I start doing something I like, so I've been busily cobbling skins together to the tune of 51 items in my workshop. I obviously don't expect them all to get in, but I did have a few that were pretty popular and are now "dying on the vine" so to speak. That being said, I'm more than happy to have had anything accepted at all and I'm having a blast making these things. I'd just like to see some of those old pieces get some love as I'm kinda proud of 'em.

**Comment by SparkleMade** [comment]
Gary, First thank you for talking to us.  I deal with consumers at work and its rough, I cannot imagine being in your shoes.  Hopefully nobody has ever thrown a brick through your window, for every turd theres 100 of us who want to have your baby.

Second, my wife has never said a good thing about video games until you posted the salt shaker.  We are in house saving mode so those pennies go a long way towards me getting that screen time without heat 🥇 

Third, I think you just need to find the middle ground, keeping in mind you will never make up for what is lost in the first week (release week of a new item).

The old pricing, I liked the way you had it going before, problem was keeping up with all the skin purchases got costly... I would know, I literally bought every skin in game up to the 99 cent drops.  At the Big Grin skin drop there were too many releases with too high a cost.  So before Big Grin drop it was $10 here $20 there... then $100+ $70+ in a week, My wife saw the emails once from the steam store and I caught her writing down the totals...  had to tear the paper out of her hand, just a disaster.

I like your planning, or the idea to increase the value as time goes on, but I also had no issue with the previous pricing, maybe the median price was just a bit too high... 3 $15 items vs 3 $5 is a big difference.  What are the spending habits of your target audience - I assume there are a lot of people who spend $5 a week, and a handful who will buy everything.  How do you cater to both demographics?

1. Keep up the limited edition $5-$20 cost specials, selling special items in sets, players love it and trend is going (glow masks, sleeping bags, bat pistol).  **Make it clear as day the item is limted edition temporary! Set the timing so everybody learns they have a set amount of time to buy L.E. items (i.e. 2 and a half weeks, enough time for a payday cycle). Can you put limited edition on the preview?**

2. Lower the median pricing, setting price points to how people spend. Guys drop 100+ 20+ and 5+ ... how do you get them to drop that extra $$$.  Right now pricing everything same is leaving money on the table, and hurting the content creators.  One example, the gold rock was $5 on the market and my Salt Shaker was $0.99...  It could be the gold rock made the content creator more money than my gun, even if my gun had higher total sales.  At $0.99 that gun left a lot of money on the table.

3. Don't rewrite the books.  The 99 cent skins are for kids who message me asking them to buy them skins "cause mom wont."  But at the same time a friend of mine mentioned he wasn't going to purchase the skin cause it was too inexpensive and "everybody was gonna have it" (can't please all the people all the time).  

A little differentiation in price would go a long way to the appearance of value, and it would support the content creators who have better textured items that guys may pay $0.50-$1 more for.  Imagine if my shaker was 2 dollars, a person paying $4 for a few skins would certainly pay an extra dollar or two for the one or two skins they valued more and you'd earn that extra dough... it also seems there is no opportunity like an update and a new launch. 

I do not like the idea of differentiation based on the quality of the item ( burlap, sleeping bag, AK) I think it should be based on the value you think the people will pay for the item.  You know the figures, look at the glow items, look at the glow sight items, look at how the black chestplate and glow sight pistol have performed.  Those two items have become staples in the game that people will pay premium for and how do you capitalize on that opportunity (without shaking up salt... yeah right).

4. Can content creators get a cut off the steam market like CSGO? Right now salt shaker is higher on the auction house and the sales are substantial.  There are guys who have very expensive items for auction who have only ever had one submission, they are missing out on a lot of residual royalties.  I of course only ask out of the creators personal interest, so I understand if you write me off from this point on.  You just miss 100% of the shots you don't take.

When I sat down I didn't intend to write this novel, sorry, weed and video gamer OCD.  Thank you and a million hugs and kisses from Wisconsin USA.

EDIT, launch the temporary edition concept (timeline and special thumbnail) with a relaunch of the blue Sandbox shirt, you could charge up to $10 bucks, it would draw tons of attention cause the demand is insane and it would make a lot of haters happy.... $5 would bring a lot of sales, but kill the market price.

**Comment by SteveDredd** [comment]
> Second, my wife has never said a good thing about video games until you posted the salt shaker. We are in house saving mode so those pennies go a long way towards me getting that screen time without heat :1st_place_medal:

I have the exact same thing going on since some of my skins got accepted. XD But on topic; this is a really good idea.

**Comment by Dripahh** [comment]
I agree with gootahba, You shoud have something like  limted edition items.
The $0.99 items are inexpensive I think its true everyone will run with them in game, thats why i dont buy the $0.99 ones myself. Items that look good shoud be more expensive than the 'boring' ones.  for example you can't put the Glowing Skull for a starter price of $0.99

**Comment by Vazul267** [comment]
Maybe this means only boring items will be accepted in the future? (lol)
I mean this week was boring enough for the 1€ price tags.
You should accept more colorful and different skins, people will get bored of the same type of skins every week.

I still think that you should accept skins in a monthly cycle and put them in crates.
Then if you don't get what you want, you could still buy them in the store for an increased price. Kinda like in the TF2 store.

**Comment by VadimSushko** [comment]
compleyely support DutchAllians idea about price categories

**Comment by raycohaex** [comment]
@Vazul267 I tend to agree, but have you seen how many skins get published every month lol, im afraid everyone will publish their stuff 3 days before they can be added

**Comment by TypicalHog** [comment]
I really, really like the idea.

**Comment by C336** [comment]
I'm by no means experienced with the player marketplace and how it behaves, but what I did notice is at no point did my skin ever drop to, or below the the price listed in the item store while it was up; and nor is it likely to as there is a cost associated with the player market and people want minimum return. 

To that extent however, exclusivity definitely seems to drive up price, as evidenced by the pulling of the 'Punishment Skull' soon after its initial release, and ludacious day 1 market prices in general. With all that said though, I am still slightly salty I didn't get the head's up that my skin would be getting the boot from the store so soon with this change.

**Comment by fikibook** [comment]
I like the idea but could you make the purchased skin to be equipped with the skin on the first spawn? Like i got ripe watermelon skin on my rock but i have to craft another rock to use it i mean come on who ever crafts a rock in this game? I got 1010 hours and i have never crafted a rock before :)

**Comment by Nevill112** [comment]
Increasing price by day is a good idea but there must be a limit to the increase, otherwise if users see that item's price increases from 1$ to 20$ they will simply buy what they want on day one and the next day se won't see any sales.

Price must also represent quality of the item produced. There's someone who spends several days working on a single item, as if it was a job, and then there's someone who lazily slaps a random texture on the item, without very basic knowledge of what they're doing. At the end work of both authors costs the same, which is not fair at all. Sell low quality items for 1$, medium quality items for 2$ and high quality for 3$ (just an example).

**Comment by StabbyMcStabface** [comment]
I kind of worry about the gradual increase as well in that, as Nevill112 said, people may just shop on week one and then say "well, I'll wait for the next round of 99 centers". Reading through all of the replies thus far, though, it seems as though everyone agrees that pricing should range from a low end to a high end.. I don't think there should be a price difference based on quality, though (as the items that make it to market should all be of at least decent enough quality to merit purchase in the first place). Maybe  random pricing between .99 cents and $3.99 and have the shelflife of each item be longer the less it costs so that the creators all see about the same amount in terms of returns. 

Something like:

.99    Items - Twelve Weeks in Store
1.99  Items - 6 Weeks in Store
3.99  Items - 3 Weeks in Store 

The more expensive items are obviously "rarer" and the cheapest items "common". Maybe have a "Sale Ends" date on each item so that people know it's going to go off-market soon, as well.
